### PR TITLE
290 request assignment

### PR DESCRIPTION
**Pull request by rkzel**
Based on 292 panel, Request Assignment is ready to be merged.

### CODE REVIEW

File: apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js [hunk 1]
```diff
@@ -1,21 +1,162 @@
...
-  rate: PropTypes.number,
-  onSubmit: PropTypes.func,
+import {
+  Field,
+  Text,
+  TextInput,
+  theme,
+  Info,
+  SafeLink,
+  IconAttention
```
Comment (Quazia) [inline apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js]: Might not be needed

File: apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js [hunk 2]
```diff
@@ -1,21 +1,162 @@
...
+} from '@aragon/ui'
+
+import { Form, FormField, FieldTitle, DateInput } from '../../Form'
```
Comment (Quazia) [inline apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js]: FieldTitle may not be needed

File: apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js [hunk 3]
```diff
@@ -1,21 +1,162 @@
...
+import { IconCalendar, IconGitHub, CheckButton } from '../../Shared'
```
Comment (Quazia) [inline apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js]: IconCalender may not be needed

File: apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js (hunk 1 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js]: IconAttention wasn't needed, the others are useful.

File: apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js (hunk 2 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js]: Removed

File: apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js (hunk 3 above)
Comment (rkzel) [reply apps/projects/app/components/Panel/RequestAssignment/RequestAssignment.js]: True, it is used only in DateInput.

**Pull request merged**